infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -108,14 +115,7 @@ private String generateNodeStartCommand(String instanceId, String rmUrlToUse, St
 
     public String generateDefaultIaasConnectorURL(String DefaultRMHostname) {
         if (nsConfig == null) {
```

Comment:
I suggest to remove the whole if block as it is useless. I prefer the mode fail fast when nsConfig is null

Reply:
Done, I've added `throw new RuntimeException(e);` when catch `ConfigurationException` during loadConfig.